Resources/Prototypes/Entities/Objects/Devices/cartridges.yml

```diff
@@ -40,6 +40,27 @@
       state: news_read
   - type: NewsReadCartridge
 
+- type: entity
+  parent: BaseItem
+  id: CrewManifestCartridge
+  name: Crew Manifest Cartridge
+  description: A program for listing your fellow crewmembers
+  components:
+    - type: Sprite
+      sprite: Objects/Devices/cartridge.rsi
+      state: cart-y
+    - type: Icon
+      sprite: Objects/Devices/cartridge.rsi
+      state: cart-y
```

Comment:
This IconComponent is necessary as it's just duplicating the sprite.

Reply:
Should I drop
```c#
- type: Icon
  sprite: Objects/Devices/cartridge.rsi
  state: cart-y
```
for all cartridges to keep them consistent?

Reply:
Yeah it's not needed.

Reply:
Dropped